.github/workflows/update-maintainers.yml

```
@@ -149,8 +155,10 @@ jobs:
 
       - name: Create new branch
         working-directory: ./community
+        env:
+          RUN_ID: ${{ github.run_id }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting

(shellcheck)